native/src/routes/dashboard/components/__tests__/Dashboard.spec.js

```diff
@@ -0,0 +1,138 @@
+  const categoryLeaf = categoriesMapModel.toArray().find(category => category.isLeaf(categoriesMapModel))
+  const language = 'de'
+  const navigation = createNavigationScreenPropMock()
+
+  const navigateToPoi = jest.fn()
+  const navigateToCategory = jest.fn()
+  const navigateToEvent = jest.fn()
+  const navigateToInternalLink = jest.fn()
+  const navigateToNews = jest.fn()
+  const navigateToOffers = jest.fn()
+  const theme = lightTheme
+
+  if (!categoryLeaf) {
+    throw Error('There should be a leaf!')
+  }
+
+  const stateView = new CategoriesRouteStateView(
+    categoryLeaf.path,
+    { [categoryLeaf.path]: categoryLeaf },
+    { [categoryLeaf.path]: [] }
+  )
+  const resourceCache = {}
+  const resourceCacheUrl = 'https://localhost:8080'
+  const t = jest.fn(key => key)
+
+  const createCityModel = (events: boolean, offers: boolean, local: boolean, tunews: boolean, pois: boolean) =>
+    new CityModel({
+      name: 'Stadt Augsburg',
+      code: 'augsburg',
+      live: true,
+      eventsEnabled: events,
+      offersEnabled: offers,
+      pushNotificationsEnabled: local,
+      tunewsEnabled: tunews,
+      poisEnabled: true,
+      sortingName: 'Augsburg',
+      prefix: 'Stadt',
+      latitude: 48.369696,
+      longitude: 10.892578,
+      aliases: {
+        Konigsbrunn: {
+          latitude: 48.267499,
+          longitude: 10.889586
+        }
+      }
+    })
+
+  const mockBuildConfig = (pois: boolean, newsStream: boolean) => {
+    const previous = buildConfig()
+    // $FlowFixMe flow is not aware that buildConfig is a mock funciton
+    buildConfig.mockImplementation(() => ({
+      ...previous,
+      featureFlags: { ...previous.featureFlags, pois, newsStream }
+    }))
+  }
+
+  const renderDashboard = (cityModel: CityModel) =>
+    <Dashboard navigation={navigation} navigateToPoi={navigateToPoi} navigateToCategory={navigateToCategory}
+               navigateToEvent={navigateToEvent} navigateToInternalLink={navigateToInternalLink}
+               navigateToNews={navigateToNews} navigateToOffers={navigateToOffers} theme={theme} language={language}
+               cityModel={cityModel} stateView={stateView} resourceCache={resourceCache}
+               resourceCacheUrl={resourceCacheUrl} t={t} />
+
+  it('should show navigation tiles if there are is at least one feature enabled', () => {
+    const cityModel = createCityModel(true, false, false, false, false)
+    const result = TestRenderer.create(renderDashboard(cityModel))
+
+    expect(() => result.root.findByType(NavigationTiles)).not.toThrowError()
+    const navigationTiles = result.root.findByType(NavigationTiles)
+    expect(navigationTiles.props.tiles.some(tile => tile.path === 'categories')).toBeTrue()
+  })
+
+  it('should not show navigation tiles if there are no features enabled', () => {
+    const cityModel = createCityModel(false, false, false, false, false)
+    const result = TestRenderer.create(renderDashboard(cityModel))
```

Comment:
In the future we should try to use the @testing-library as already started in other tests.

Reply:
You're definitely right! I somehow thought this is the same library :facepalm: